src/Shirshu.php

```diff
@@ -46,18 +46,12 @@ public static function getInstance(){
         return self::$instance;
     }
 
-    /**
-     * Create array of supported plugins that Shirshu currently supports
-     */
-    private function defineSupportedPlugins(){
-        $this->supportedPlugins = array('Contact Form 7', 'Formidable Forms', 'Ninja Forms', 'WooCommerce', 'WPForms', 'WPForms Lite');
-    }
-
     /**
      * Shirshu constructor
      */
     private function __construct(){
-        $this->defineSupportedPlugins();
+        $this->supportedPlugins = array('Contact Form 7', 'Formidable Forms', 'Ninja Forms', 'WooCommerce', 'WPForms',
+            'WPForms Lite');
         $this->injector = new Injector($this->supportedPlugins);
```

Comment:
Please rename MeasurementCodeInjector. Initially, I thought that Injector refers to a dependency injection framework.